assets/scripts/store/reducers/settings.js

```diff
@@ -0,0 +1,28 @@
+import { SET_USER_SETTINGS } from '../actions'
+// Note: turning this constant on runs too much side-effect code too early.
```

Comment:
Oh, good questions. I should make this clearer. The comment is referring to the line below (not the one above) where I import `NEW_STREET_DEFAULT` from [`../../streets/creation`](https://github.com/streetmix/streetmix/blob/master/assets/scripts/streets/creation.js), and use it when defining `initialState`. The problem is that I only need the constant, but importing that module here causes it to run, and causes a chain reaction where every other import in its dependency tree is also run, and some things end up running when it's not supposed to yet. I don't remember exactly which code runs too early, but it's getting triggered on the import of this particular file.

I could make this comment clearer and move on, or perhaps we do something else, like export constants from a module that doesn't run other code?